sputnikdao2/src/types.rs

```diff
@@ -1,11 +1,14 @@
 use near_sdk::borsh::{self, BorshDeserialize, BorshSerialize};
 use near_sdk::json_types::Base64VecU8;
 use near_sdk::serde::{Deserialize, Serialize};
-use near_sdk::{Balance, Gas};
+use near_sdk::{AccountId, Balance, Gas};
 
-/// Account ID used for $NEAR.
-pub const BASE_TOKEN: &str = "";
+/// Account ID used for $NEAR in near-sdk v3.
+/// Need to keep it around for backward compatibility.
+pub const OLD_BASE_TOKEN: &str = "";
 
+/// Account ID that represents a token in near-sdk v3.
+/// Need to keep it around for backward compatibility.
 pub type OldAccountId = String;
```

Comment:
@ctindogaru consider `V2AccountId` or similar to denote when and what this is for, so future can be removed.

Reply:
V2 might be confusing because this is a near sdk issue and people will not know if we're referring to the near sdk version or the sputnik version.

Also, the upgrade documentation advise to prefix everything with Old. See https://www.near-sdk.io/upgrading/production-basics